### PR TITLE
[UI] Show more appropriate fields when viewing a pool

### PR DESCRIPTION
**Pull request by thomasdavis**

### CODE REVIEW

File: ui/app/src/components/poolList/SinglePool.vue [hunk 1]
```diff
@@ -54,33 +61,44 @@ export default defineComponent({
       const t = toToken.value;
       return t.imageUrl;
     });
-    const toValue = computed(() => accountPool?.value?.pool.amounts[0].toFixed(DECIMALS));
+
+    const toTotalValue = computed(() => accountPool?.value?.pool.amounts[0].toFixed(DECIMALS));
```
Comment (ryardley) [inline ui/app/src/components/poolList/SinglePool.vue:65]: I am wondering with this number formatting does it make sense to throw this all in a function now or wait until later?

Comment (thomasdavis) [reply ui/app/src/components/poolList/SinglePool.vue:65]: From our call this morning, I think someone wanted to rework the concept of to/from in most places, maybe it is better left until that refactor?

**Review by ryardley — APPROVED** [review]
Not sure if you wanna follow this with a cleanup PR with sorting out decimals in a function?

File: ui/app/src/components/poolList/SinglePool.vue (hunk 1 above)
Comment (ryardley) [reply ui/app/src/components/poolList/SinglePool.vue:65]: Fair enough

Comment (ryardley) [reply ui/app/src/components/poolList/SinglePool.vue:65]: This file can be simplified pretty majorly

**Comment by 59023g** [comment]
👍

**Pull request merged**